tests/test_gridsearch/test_base.py

```diff
@@ -0,0 +1,133 @@
+# # tests.test_gridsearch.test_base.py
+# # Test the GridSearchColorPlot (standard and quick visualizers).
+# #
+# # Author:   Tan Tran
+# # Created:  Sat Aug 29 12:00:00 2020 -0400
+# #
+# # Copyright (C) 2020 The scikit-yb developers
+# # For license information, see LICENSE.txt
+# #
+
+"""
+Test the GridSearchColorPlot visualizer.
+"""
+
+# ##########################################################################
+# ## Imports
+# ##########################################################################
+
+import pytest
+
+from tests.base import VisualTestCase
+from tests.fixtures import Dataset, Split
```

Comment:
Looks like the `Split` import is not used in the file; would you mind removing it to resolve the unused imports errors on Travis?